c2cgeoform/views/abstract_views.py

```diff
@@ -174,14 +174,21 @@ def __init__(self, request):
         self._appstruct = None
         self._obj = None
 
+        self._request.response.cache_control.no_cache = True
+        self._request.response.cache_control.max_age = 0
+        self._request.response.cache_control.private = True
+
     def index(self):
+        self._request.response.cache_control.no_cache = False
+        self._request.response.cache_control.max_age = 3600  # one hour
```

Comment:
Why not use no_cache for all services? 1h looks arbitrary.

Reply:
Because the tables format don't change on standard usage, and one hour is already use in the static views :-)